values.yaml

```diff
@@ -63,10 +63,10 @@ namespaces:
 - name: doc-checking-staging
   owner: alphagov
   repository: doc-checking
-  branch: master
+  branch: OGD-253/make-pipeline-push-to-harbour
   path: ci/staging
   permittedRolesRegex: "^$"
-  requiredApprovalCount: 2
+  requiredApprovalCount: 0
```

Comment:
Why is the approval count being set to zero?

Reply:
Even though the required approvers restriction applies only to the `master` branch, Concourse reads it no matter what branch it's deployed from. So, when people PR from another branch into `OGD-253/make-pipeline-push-to-harbour`, they don't _need_ approvers, but if we don't set this to 0 then Concourse will complain and refuse to deploy the changes.

Reply:
We want to be able to perform fast iteration whilst we're still developing the pipeline. Ideally, we'd use GSP local for this sort of iterative development, but it's a bit broken at the moment 🤷‍♂ 

This is a non-prod pipeline, so I don't think it is a problem. We're currently doing the same thing in our build pipeline. If you look further down the file, we wanted to set this for staging, but it looks as if it is getting overwritten by the legacy settings.